Source/Core/Common/CommonPaths.h

```diff
@@ -7,6 +7,7 @@
 // Directory separators, do we need this?
 #define DIR_SEP "/"
 #define DIR_SEP_CHR '/'
+#define DIR_SEP_WIN32 "\\"
```

Comment:
I'm extremely skeptical this is even necessary.

Reply:
Well, I had 5 cases (5 files) and there's more characters that the two which already exist, I'll wait a bit for what others think about this.

Reply:
Please remove this.